standard-libraries/math.3.mr

```diff
@@ -0,0 +1,45 @@
+## Math functions for integers
```

Comment:
This is temporary, module-system syntax is not decided yet (#58).
In the future they will be under `math` module.